R/geometry.R

```diff
@@ -779,7 +779,7 @@ as.Lines.raw <- function(cl, ID = " ") {
 #'
 #' @examples
 #' \dontrun{
-#' if (require.nowarnings("INLA") && require("fmesher")) {
+#' if (require("INLA") && require("fmesher")) {
```

Comment:
require.nowarnings() is meant to be used whenever requiring INLA. If INLA isn't required, remove the require() call entirely.